test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,76 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class UtilsTest {
+    private static final Object NULL_OBJECT = null;
```

Comment:
Oh, it seems like this is another icky Java thing :-)

Yeah, if we want to call `isAnyNull(null)`, we need to make it explicit by using `isAnyNull((Object)null)`.